extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -75,6 +76,31 @@ public function renderInfobox( array $infoboxdata, $theme, $layout ) {
 			$infoboxHtmlContent = $this->renderInfoboxHero( $heroData ) . $infoboxHtmlContent;
 		}
 
+		$galleryData = Array (
+			Array (
+				'url' => 'http://vignette.wikia-dev.com/visualeditor/images/b/b8/Challenger.jpg/revision/latest?cb=20140626002212',
+				'name' => 'Challenger.jpg',
+				'key' => 'Challenger.jpg',
+				'alt' => '',
+				'caption' => 'This is a caption'
+			),
+			Array (
+				'url' => 'http://vignette.wikia-dev.com/visualeditor/images/1/1d/Challenger-1.jpg/revision/latest?cb=20140626002317',
+				'name' => 'Challenger-1.jpg',
+				'key' => 'Challenger-1.jpg',
+				'alt' => '',
+				'caption' => 'This is a caption for Challenger 1'
+			),
+			Array (
+				'url' => 'http://vignette.wikia-dev.com/visualeditor/images/4/41/Challenger-0.jpg/revision/latest?cb=20140626002239',
```

Comment:
Please add a couple of images to cover use cases of:
* image in portrait orientation
* small image (narrower than 300px)
* images with aspect ratios typical for use cases on some wikis:
** 1:2 (happens for characters, melee weapons)
** 2:1 (i.e. firearms)